tools/run-tests.py

```diff
@@ -323,6 +324,12 @@ def run_buildoption_test(options):
 
     return ret
 
+def run_coverity_scan(options):
```

Comment:
Why is this change needed? Why `build_command: "tools/build.py --clean"` in travis.yml at line 37 isn't enough?